gui/src/vmm/cpu/mod.rs

```diff
@@ -520,19 +534,54 @@ impl<H: Hypervisor, S: Screen> ThreadExtraInfo for CpuManager<H, S> {
 
 impl<H: Hypervisor, S: Screen> MultiThreadResume for CpuManager<H, S> {
     fn resume(&mut self) -> Result<(), Self::Error> {
-        todo!()
+        self.release();
+
+        Ok(())
     }
 
     fn clear_resume_actions(&mut self) -> Result<(), Self::Error> {
-        todo!()
+        for cpu in &mut self.cpus {
+            cpu.resume_action = None;
+        }
+
+        Ok(())
     }
 
     fn set_resume_action_continue(
         &mut self,
         tid: Tid,
         signal: Option<Signal>,
     ) -> Result<(), Self::Error> {
-        todo!()
+        let cpu = self
+            .cpus
+            .get_mut(tid.get() - 1)
+            .ok_or(GdbError::CpuNotFound)?;
+
+        cpu.resume_action = Some(ResumeAction::Continue);
+
+        Ok(())
+    }
+
+    #[inline(always)]
+    fn support_single_step(&mut self) -> Option<MultiThreadSingleStepOps<'_, Self>> {
+        Some(self)
+    }
```

Comment:
This should requires additional works on the hypervisor (e.g. `KVM_GUESTDBG_SINGLESTEP` on KVM). I'm not sure if the other hypervisor support this. If not we cannot enable this.

Reply:
I'll remove it for now.